spec/.straight/config.yml

```diff
@@ -45,6 +45,8 @@ blockchain_adapters:
 
 insight_url: "https://insight.mycelium.com/api"
 insight_test_url: "https://test-insight.bitpay.com"
+
+insight_twebsocket_url: "wss://insight.mycelium.com"
```

Comment:
it's typo. 
correct should be `insight_websocket_url`
`websocket` in naming is more concrete and explicit than just `socket`